screenpipe-vision/build.rs

```diff
@@ -20,29 +20,39 @@ fn main() {
         let binary_path = bin_path.join(binary_name);
 
         println!("cargo:rerun-if-changed=src/ui_monitoring_macos.swift");
-        
+
         let status = Command::new("swiftc")
-            .args(&[
+            .args([
                 "-O",
                 "-whole-module-optimization",
                 "-enforce-exclusivity=unchecked",
-                "-num-threads", "8",
-                "-target", if cfg!(target_arch = "aarch64") { 
-                    "arm64-apple-macos11.0" 
-                } else { 
-                    "x86_64-apple-macos11.0" 
+                "-num-threads",
+                "8",
+                "-target",
+                if cfg!(target_arch = "aarch64") {
+                    "arm64-apple-macos11.0"
+                } else {
+                    "x86_64-apple-macos11.0"
                 },
-                "-o", binary_path.to_str().unwrap(),
+                "-o",
+                binary_path.to_str().unwrap(),
                 "src/ui_monitoring_macos.swift",
-                "-framework", "Cocoa",
-                "-framework", "ApplicationServices",
-                "-framework", "Foundation",
+                "-framework",
+                "Cocoa",
+                "-framework",
+                "ApplicationServices",
+                "-framework",
+                "Foundation",
             ])
             .status()
             .expect("failed to compile Swift executable");
-            
+
         if !status.success() {
             panic!("failed to compile ui_monitor executable");
         }
+
+        let bin_path = PathBuf::from(&manifest_dir).join("bin");
```

Comment:
?

Reply:
I added this because in development, the ui monitor was being built with a different name and I had to manually copy it to use the changes.